src/app.js

```
@@ -1,12 +1,13 @@
-  { id: 1, name: '商品1', price: 99.99 },
-  { id: 2, name: '商品2', price: 199.99 },
-  { id: 3, name: '商品3', price: 299.99 }
+  { id: 1, name: '商品1', price: 99.99, discount: 0.8 },
+  { id: 2, name: '商品2', price: 199.99, discount: 0.7 },
+  { id: 3, name: '商品3', price: 299.99, discount: 0.9 }
 ];
 
 class ShoppingApp {
   constructor() {
     this.cart = [];
+    this.activityEndTime = new Date(Date.now() + 2 * 60 * 60 * 1000);
```

ESLint

[error] Extra semicolon.

(semi)